### DISCUSSION TITLE
Recommended way to create a policy not attached to a role

### DISCUSSION THREAD

**Discussion by fsanzdev** [gruntwork-io/knowledge-base#585] [Help]
Hi, 

I need to create a policy not attached to a role (it will be attached from an external script not integrated in TF). Is there a recommended way to create it? I can't figure out a proper way to do it using the security modules. 

Thanks!

---

<ins datetime="2022-11-08T15:22:55Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109570">Tracked in ticket #109570</a></p>
</ins>

**Comment by autero1** [comment]
Hi, we don't have a specific module for creating arbitrary policies. You can best accomplish this by creating a wrapper module for [`aws_iam_policy` resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_policy).
